test/CSharp/CodeCracker.Test/Design/CatchEmptyTests.cs

```diff
@@ -44,7 +45,7 @@ namespace ConsoleApplication1
     {
         class TypeName
         {
-            public async Task Foo()
+            public async {|CS0246:Task|} {|CS0161:{|CS1983:Foo|}|}()
```

Comment:
📝 These compiler errors are due to a missing import of `System.Threading.Tasks`.